### PR TITLE
Adding deploying redis section

### PR DESCRIPTION
**Pull request by g-murray**
[AAP-28163 ](https://issues.redhat.com/browse/AAP-28163)

- Adding a section on how to deploy Clustered Redis. 
- Updating Chapter four assembly to include this new section

### CODE REVIEW

File: downstream/modules/platform/proc-operator-deploy-redis.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+= Deploying clustered Redis on {OperatorPlatform}
+
+When you create an {PlatformNameShort} instance through the {OperatorPlatform}, standalone Redis is assigned by default. 
+To deploy clustered Redis, see the following procedure.
```
Comment (ianf77) [inline downstream/modules/platform/proc-operator-deploy-redis.adoc:4]: use the following procedure?

File: downstream/modules/platform/proc-operator-deploy-redis.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+//Add a link to the section when ready
+For more information about Redis, refer to Caching and queueing system in the _Planning_ _your_ _installation_ _guide_.
```
Comment (ianf77) [inline downstream/modules/platform/proc-operator-deploy-redis.adoc:7]: , see <link>[Caching and queuing system] in the _Planning your installation guide_

Comment (ariordan-redhat) [inline downstream/modules/platform/proc-operator-deploy-redis.adoc:7]:
```suggestion
For more information about Redis, refer to Caching and queueing system in the _Planning your installation_ guide.
```

File: downstream/modules/platform/proc-operator-deploy-redis.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* You have installed an {OperatorPlatform} deployment.
+
+.Procedure
+. Log in to {OCP}. 
+. Navigate to menu:Operators[Installed Operators].
+. Select your {OperatorPlatform} deployment.
+. Select the *Details* tab. 
+. On the *{PlatformNameShort}* tile click btn:[Create instance].
+.. For existing instances, you can edit the YAML view by clicking the {MoreActionsIcon} icon and then btn:[Edit AnsibleAutomationPlatform].
+.. Edit the *redis_mode* value to "clustered".
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-operator-deploy-redis.adoc:19]:
```suggestion
.. Change the *redis_mode* value to "clustered".
```

**Review by ianf77 — COMMENTED** [review]
A couple of comments.

**Review by ianf77 — APPROVED** [review]
LGTM

**Review by ariordan-redhat — COMMENTED** [review]
LGTM

**Pull request merged**